src/components/Dependencies/Dependencies.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import Accordion from "@mui/material/Accordion";
+import Box from "@mui/material/Box";
+import CircularProgress from "@mui/material/CircularProgress";
+import InfiniteScroll from "react-infinite-scroll-component";
+import {
+  StyledAccordionExpandIcon,
+  StyledAccordionDetails,
+  StyledAccordionSummary,
+  StyledAccordionTitle
+} from "src/styles";
+import DependenciesItem from "./DependenciesItem";
+import { Dependency } from "src/common/models";
+
+interface IDependenciesProps {
+  dependencies: Dependency[];
+}
+
+const Dependencies = ({ dependencies }: IDependenciesProps) => {
+  const [list, setList] = useState(dependencies);
+  const listLength = list.length;
+
+  const nextDependencies = [
+    {
+      id: Math.random(),
+      channel: {
+        id: 2,
+        name: "https://conda.anaconda.org/conda-forge",
+        last_update: null
+      },
+      build: "pyhd8ed1ab_0",
+      license: "Apache-2.0",
+      sha256:
+        "4da0fe03babc950532513e9165dbc337a663880352392f496992776608dd77ca",
+      name: "asttokens",
+      version: "2.0.5",
+      summary:
+        "The asttokens module annotates Python abstract syntax trees (ASTs) with the positions of tokens and text in the source code that generated them."
+    },
+    {
+      id: Math.random(),
+      channel: {
+        id: 2,
+        name: "https://conda.anaconda.org/conda-forge",
+        last_update: null
+      },
+      build: "pyh9f0ad1d_0",
+      license: "BSD-3-Clause",
+      sha256:
+        "ee62d6434090c1327a48551734e06bd10e65a64ef7f3b6e68719500dab0e42b9",
+      name: "backcall",
+      version: "0.2.0",
+      summary: "Specifications for callback functions passed in to an API"
+    },
+    {
+      id: Math.random(),
+      channel: {
+        id: 2,
+        name: "https://conda.anaconda.org/conda-forge",
+        last_update: null
+      },
+      build: "pyh9f0ad1d_0",
+      license: "BSD-3-Clause",
+      sha256:
+        "ee62d6434090c1327a48551734e06bd10e65a64ef7f3b6e68719500dab0e42b9",
+      name: "backcall",
+      version: "0.2.0",
+      summary: "Specifications for callback functions passed in to an API"
+    }
+  ];
+
+  const addDependencies = () => {
+    if (list.length <= 15) {
+      setTimeout(() => {
```

Comment:
This is just to simulate an API call, I'll remove this as soon as we start using the API